### PR TITLE
Installing the bigdesk monitoring plugin.

### PR DESCRIPTION
**Pull request by e0d**
Bigdesk is a nice monitoring dashboard that provides a lot of detail about cluster health, etc.

https://raw.github.com/lukas-vlcek/bigdesk/master/bigdesk-2.0.0-SNAPSHOT.jpg

### CODE REVIEW

File: playbooks/roles/elasticsearch/tasks/main.yml [hunk 1]
```diff
@@ -63,6 +63,18 @@
     src=edx/etc/elasticsearch/elasticsearch.yml.j2 dest={{ elasticsearch_cfg_dir }}/elasticsearch.yml
     mode=0744
   when: ELASTICSEARCH_CLUSTERED
-    
+
+  # Plugin installation fails hard when the plugin already
+  # exists. This is problematic of this is upgraded.
```
Comment (jarv) [inline playbooks/roles/elasticsearch/tasks/main.yml]: typo

File: playbooks/roles/elasticsearch/tasks/main.yml [hunk 2]
```diff
@@ -63,6 +63,18 @@
...
+- name: check if the bigdesk plugin is installed
+  command: test -d /usr/share/elasticsearch/plugins/bigdesk
+  ignore_errors: true
+  register: bigdesk_present
```
Comment (jarv) [inline playbooks/roles/elasticsearch/tasks/main.yml]: would prefer to use the stat module instead, so it doesn't show a red error when it fails which will be skipped with ignore_errors but is still ugly.

**Comment by feanil** [comment]
looks good.

**Comment by jarv** [comment]
:+1:

**Pull request merged**